client/src/components/CardSearchResult.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
+import Button from '../components/Button';
+
+const Container = styled.article`
+  display: flex;
+  flex-flow: column nowrap;
+  min-width: 330px;
+  height: 566px;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.4);
+`;
+const ImageBox = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 243px;
+  width: 100%;
+  border-radius: 5px 5px 0 0;
+`;
+const Image = styled.img`
+  object-fit: cover;
+  height: 233px;
+`;
+
+const InfoBox = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-around;
+  height: 324px;
+  width: 100%;
+  border-radius: 0 0 5px 5px;
+  background-color: ${colors.bgSecondary};
+`;
+
+const Title = styled.h2`
+  align-self: flex-start;
+  font-family: SF Pro Rounded Regular;
+  font-size: 35px;
+  margin: 0 0 4px 18px;
+  color: ${colors.textPrimary};
+`;
+
+const Info = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+
+const InfoItem = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  margin: 6px 0 0 0;
+  font-family: SF Pro Rounded Regular;
+`;
+
+const InfoData = styled.span`
+  color: ${colors.textSecondary};
+  font-size: 28px;
+`;
+
+const InfoLabel = styled.span`
+  color: ${colors.textInfo};
+  font-size: 18px;
+`;
+
+const AddButton = styled(Button)`
+  align-self: center;
+  width: 230px;
+`;
+
+export const CardSearchResult = ({ details }) => {
+  return (
+    <Container>
+      <ImageBox>
+        <Image src={details.img} />
+      </ImageBox>
+      <InfoBox key={details.id}>
+        <Title>{details.title}</Title>
+        <Info>
+          <InfoItem>
+            <InfoLabel>#Item</InfoLabel>
+            <InfoData>{details.item}</InfoData>
+          </InfoItem>
+          <InfoItem>
+            <InfoLabel>Theme</InfoLabel>
+            <InfoData>{details.theme}</InfoData>
+          </InfoItem>
+          <InfoItem>
+            <InfoLabel>Year</InfoLabel>
+            <InfoData>{details.year}</InfoData>
+          </InfoItem>
+        </Info>
```

Comment:
you could move this to a separate component, maybe one that can receive `InfoItems` and then renders its three items based on what props are passed, or even make it more dynamic and let it render an array of `Items` and divides them into rows based on how much horizontal space it has.

Try to make sense of this, if you want to chat let me know :D

Reply:
👍🏻